react/components/molecules/key-value-placeholder/key-value-placeholder.js

```diff
@@ -0,0 +1,82 @@
+import React, { PureComponent } from "react";
+import { StyleSheet, View } from "react-native";
+
+import LinearGradient from "react-native-linear-gradient";
+
+import PropTypes from "prop-types";
+
+export class KeyValuePlaceholder extends PureComponent {
+    static get propTypes() {
+        return {
+            numberOfLines: PropTypes.number,
+            colors: PropTypes.arrayOf(PropTypes.string).isRequired,
+            locations: PropTypes.arrayOf(PropTypes.number).isRequired
+        };
+    }
+
+    static get defaultProps() {
+        return {
+            numberOfLines: 1
+        };
+    }
+
+    _renderLines() {
+        const componentsToReturn = [];
+
+        for (let index = 0; index < this.props.numberOfLines; index++) {
+            componentsToReturn.push(
+                <LinearGradient
+                    key={index}
+                    style={styles.value}
+                    angle={90}
+                    useAngle={true}
+                    colors={this.props.colors}
+                    locations={this.props.locations}
+                />
+            );
+        }
+        return componentsToReturn;
+    }
+
+    render() {
+        return (
+            <View style={styles.sectionViewPlaceholder}>
+                <View style={styles.textContainer}>
+                    <LinearGradient
+                        style={styles.key}
+                        angle={90}
+                        useAngle={true}
+                        colors={this.props.colors}
+                        locations={this.props.locations}
+                    />
+                    {this._renderLines()}

```

Comment:
Take a look at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill for a slightly shorter way to do this.

Reply:
Has per our conversation this suggestion has no effect now.